app/components/chart/daily.js

```diff
@@ -165,6 +165,7 @@ var Daily = React.createClass({
   },
   componentDidMount: function() {
     const dData = this.props.patientData.diabetesData;
+    const fNameUnits = this.props.bgPrefs.bgUnits.replace('/', '').toLowerCase();
```

Comment:
what is the `f` in this - don't you think we should just call it what it is ? i.e. in general I think we should rather than have non-standard abbreviations :)

Reply:
Yeah, I agree, bad naming here.  It's just to represent the `units` in the filename for the downloaded data. (so `fileNameUnits`... but I'll change it to just `units`)